tools/omicron-install-dendrite-mac.sh

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env bash
+
+# Updated from https://gist.github.com/david-crespo/d8aa7ea1afb877ff585e6ad90fc5bc2c.
+# Installs: yq, gh
```

Comment:
this isn't true, they're installed in the main brew install. you could say it relies on them though